### ISSUE TITLE
Where are the loss function $L_1$ and $L_2$?

### ISSUE THREAD

**Issue by Aris-z** [xufangzhi/ENVISIONS#2]
Hi, I note that you use two new optimized loss functions for SFT and RL-free in your paper, but I can't find them in your code. It seems that you use the loss function the same as LLAMA 2. This confuses me and I would appreciate it if you'd like to explain it.
Thanks.

**Comment by xufangzhi** [comment]
Hi, both L1 and L2 are SFT-style loss function (i.e., predicting next token). They differ in the input and output format of the training data. Therefore, we construct different sets of data for the implementation of L1 and L2. Please refer to the script like ``organize_preference_data_llama2chat.py``.

**Comment by Aris-z** [comment]
Excuse me, when I am executing the initial step, I find that no data is filtered out when the generated data pass the first label-preference.py code, which directly lead to an error in the subsequent open-instruct model training steps. What might be the reason for this?
The log is below:
![image](https://github.com/user-attachments/assets/fcb9628b-a0da-48e2-8c75-ab1ab83a496b)
